src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+module.exports = {
```

Comment:
No need to modify the other files.  We plan to run through the entire codebase before Kibana 5 to update the majority of the existing commonjs module stuff.